### PR TITLE
Task: Refactoring

### LINKED ISSUE #31: [Task] Refactoring
## Tasks:
- [x] create feature branch
- [x] sort App imports
- [x] move successMessage from App: useEffect to AddBook with state var & function for successMessage for AddBookForm
- [x] extract formatData function in utils
- [x] delete console.logs
- [x] extract code that doesn't belong to get request and put it in handleSubmit function
- [x] extract getBook in services
- [x] move FileReader and bookcover from App to AddBookForm
- [x] change history
- [x] Rename StartAddBook in SearchViaISBN (adjust components, tests, stories etc.)

### PR DESCRIPTION
**Pull request by mbosselmann**
This resolves #31 

I did refactor my code for better readability and better code! :-)

- Sort imports
- Extract functions and get request
- Remove unnecessary code
- Move functions and state to components
- Change routing in App
- Rename StartAddBook in SearchViaISBN

### CODE REVIEW

**Review by martinhaiss — COMMENTED** [review]
Excellent job! 👍 I'm quite impressed how you streamlined your code and also a bit embarrassed by myself that, despite tripple checking, I have no suggestions or corrections for you.

**Comment by mbosselmann** [comment]
Thank you, Martin! 💐

File: src/App.js [hunk 1]
```diff
@@ -1,155 +1,94 @@
...
+      )
+    } else {
+      return <Redirect to="/" />
+    }
+  } else {
+    return (
+      <AppContainer>
+        <Switch>
+          <Route exact path="/">
+            <HomeScreen onHandleSetUsername={handleSetUsername} />
```
Comment (macfrei) [inline src/App.js:55]: I think it's a bit harder to read with the `if...else`. Maybe we can have a look together next week?

Comment (mbosselmann) [reply src/App.js:55]: Sure! :-) Until then I will think about a better solution! 👍

**Pull request merged**